### ISSUE TITLE
Info about features in the compact show

### ISSUE THREAD

**Issue by CarloLucibello** [CarloLucibello/GraphNeuralNetworks.jl#169]
Right now features' information is discarded when compact printing (when printing a vector of graphs):
```julia
julia> [rand_graph(10, 20, ndata=rand(2, 10)) for i=1:5]
5-element Vector{GNNGraph{Tuple{Vector{Int64}, Vector{Int64}, Nothing}}}:
 GNNGraph(10, 20)
 GNNGraph(10, 20)
 GNNGraph(10, 20)
 GNNGraph(10, 20)
 GNNGraph(10, 20)
``` 
We could print instead at least the size of the arrays.

**Comment by aurorarossi** [comment]
What do you think about something like the following?
```julia
julia> [rand_graph(10,20,ndata=rand(5,10),edata=rand(2,20),gdata=rand(1,1)) for i in 1:5]
5-element Vector{GNNGraph{Tuple{Vector{Int64}, Vector{Int64}, Nothing}}}:
 GNNGraph(10, 20) x: 5×10 e: 2×20 u: 1×1
 GNNGraph(10, 20) x: 5×10 e: 2×20 u: 1×1
 GNNGraph(10, 20) x: 5×10 e: 2×20 u: 1×1
 GNNGraph(10, 20) x: 5×10 e: 2×20 u: 1×1
 GNNGraph(10, 20) x: 5×10 e: 2×20 u: 1×1
```
where x,e,u are the default names of ndata, edata and gdata, respectively.

**Comment by CarloLucibello** [comment]
Seems reasonable. What do we do when multiple tensors are present? What do you think about this:
```julia
julia> [
  rand_graph(10,20),
  rand_graph(10,20,ndata=rand(5,10)),
  rand_graph(10,20, ndata=(a=rand(5,10),)],
  rand_graph(10,20,ndata=rand(5,10),edata=rand(2,20)),
  rand_graph(10,20,ndata=rand(5,10),gdata=rand(1,1)),  
  rand_graph(10,20,ndata=rand(5,10),edata=rand(2,20),gdata=rand(1,1)),
  rand_graph(10,20,ndata=(a=rand(5,10), b=rand(3,10)), edata=rand(2,20)]
...
GNNGraph(10, 20) with no data
GNNGraph(10, 20) with x: 5×10 data
GNNGraph(10, 20) with a: 5×10 data
GNNGraph(10, 20) with x: 5×10, e: 2×20 data
GNNGraph(10, 20) with x: 5×10, , u: 1×1 data
GNNGraph(10, 20) with x: 5×10, e: 2×20, u: 1×1 data
GNNGraph(10, 20) with (a: 5×10, b: 3x10), e: 2×20 data
```

**Comment by CarloLucibello** [comment]
closed by #234
